src/Components/Color/ColorCard.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import "./Color.css";
+import ColorForm from "./ColorForm";
+
+/* functional component named ColorCard */
+/* receives a single prop called color, which is an object */
+
+export default function ColorCard({ color, onDelete, onUpdate }) {
+  /* set State for the EDIT-button: track if the color-card is in edit mode */
+  const [isEditing, setIsEditing] = useState(false);
+
+  /* handler function to delete the color-card */
+  const handleDeleteClick = () => {
+    const isConfirmed = window.confirm("Really delete?");
+
+    if (isConfirmed) {
+      onDelete(color.id);
+    }
+  };
```

Comment:
I know this is part of the previous issue but I suggest you to have a look at it again after finishing the edit feature. 
In the issue, you can see in the video that you actually should not use the `window.confirm` but instead a custom confirmation message. You can achieve this similar to the edit function 👉🏼 You can implement a state, which tracks if the user clicked on the delete button (with a boolean: `true`/`false` same as with `isEditing`). If the user clicked on the delete button, the custom confirmation message should appear, where the user can confirm or aboard the deletion.

Reply:
you could not know but in yesterdays session we went through this browser feature together with Felix who introduced this solution as well. But of course it is not the same as in the video but I guess it still meets the Acceptance Criteria.